src/trainer.py

```diff
@@ -67,6 +63,7 @@ def train_epoch(self, op, loss_f):
       epoch_losses.append(loss.item())
       loss.backward()
       op.step()
+      print(f"batch {i}/{len(self.train_data)}\tloss: {loss}", end="\r")
```

Comment:
out of curiosity, any reason to use print instead of logging, I suppose logging is the best practice? 
We don't need to change it anyway, both works fine for non-production

Reply:
Mainly because I'm counting on wandb to log the output. It will capture all the print statements in an output.log file.